app/models/middleware_server_group.rb

```diff
@@ -2,7 +2,6 @@ class MiddlewareServerGroup < ApplicationRecord
   include NewWithTypeStiMixin
```

Comment:
Same question, is this still needed

Reply:
Also https://github.com/ManageIQ/manageiq/pull/19378/files#diff-308561b17c3dded5ba47c4d872a670fdR5 the middleware_deployments model is being deleted so it doesn't make sense to keep the assoc if for some reason we keep this model

Reply:
Removed.